src/scene/NTEScene.py

```
@@ -30,3 +33,38 @@ def in_team(self, fun):
+            self._ocr_warm_up = True
+            try:
+                all_tasks = og.executor.get_all_tasks()
+                if all_tasks and hasattr(all_tasks[0], "ocr"):
+                    logger.info("Warming up default OCR...")
+                    all_tasks[0].ocr(frame=white_frame)
+                
+                self.make_bg_ocr()
+                all_tasks[0].ocr(frame=white_frame, lib="bg_onnx_ocr")
+
+                logger.info("OCR initialization finished.")
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)